authn.js

```diff
@@ -212,10 +212,7 @@ function setup(app) {
   // Provide the client-side app with info about the current user
   app.route("/whoami").get((req, res) => {
```

Comment:
Can this be done simply via `.get((req, res) => res.redirect('/users'))`? (I haven't tested, but think this should work)